packages/server/comms-api/routes/mail.go

```diff
@@ -1,170 +1,111 @@
 package routes
 
 import (
+	"errors"
 	"fmt"
 	"github.com/DusanKasan/parsemail"
 	"github.com/gin-gonic/gin"
 	c "github.com/openline-ai/openline-customer-os/packages/server/comms-api/config"
+	"github.com/openline-ai/openline-customer-os/packages/server/comms-api/model"
 	s "github.com/openline-ai/openline-customer-os/packages/server/comms-api/service"
-	"golang.org/x/net/context"
-	"google.golang.org/grpc/status"
 	"log"
 	"net/http"
-	"net/mail"
 	"strings"
-	//pbOasis "openline-ai/oasis-api/proto"
-	//"strings"
 )
 
-type MailPostRequest struct {
-	Sender     string `json:"sender"`
-	RawMessage string `json:"rawMessage"`
-	Subject    string `json:"subject"`
-	ApiKey     string `json:"api-key"`
-	Tenant     string `json:"X-Openline-TENANT"`
-}
+func addMailRoutes(conf *c.Config, rg *gin.RouterGroup, mailService *s.MailService) {
+	rg.POST("/mail/send", func(c *gin.Context) {
```

Comment:
There should be an api key check for sending e-mail as well

Reply:
nope. Identity is passed